### PR TITLE
Removes RNG chance to fail to emag an APC, along with the emag delay

### PR DESCRIPTION
**Pull request by Iamgoofball**
literally why was this a thing

see the pr that removes it from borgs for reasoning

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -581,13 +581,10 @@
 		else
 			flick("apc-spark", src)
 			if (do_after(user,6))
```
Comment (Fox-McCloud) [inline code/modules/power/apc.dm]: do_after, why?

Comment (Iamgoofball) [inline code/modules/power/apc.dm]: idfk why this is even there

i toyed with removing it but decided eh

Comment (Fox-McCloud) [inline code/modules/power/apc.dm]: I'd just remove it, especially with the do_after() PR.

Comment (Iamgoofball) [inline code/modules/power/apc.dm]: yeah sure why not

**Comment by MrPerson** [comment]
I don't think anybody could possibly argue with this.

**Comment by Iamgoofball** [comment]
why was it even prob(50) in the first place too

literally what was the coder smoking here

**Comment by Fox-McCloud** [comment]
It dates alllll the way back to 4407; so does the retarded borg emagging RNG.

**Comment by PKPenguin321** [comment]
:+1: to this, you could literally bypass it anyways by spam clicking the APC three or four times anyways

**Comment by TheVekter** [comment]
@Iamgoofball Do you think we should remove borg emag RNG too? I've always thought that was plenty retarded. (It's gotten me killed in a few instances, too.)

**Comment by Iamgoofball** [comment]
I already have a pr up for that

**Pull request merged**